install/error-handling.sh

```diff
@@ -15,9 +41,30 @@ cleanup () {
     return 0;
   fi
   DID_CLEAN_UP=1
-
+  local retcode=$?
+  local cmd="${BASH_COMMAND}"
   if [[ "$1" != "EXIT" ]]; then
-    echo "An error occurred, caught SIG$1 on line $2";
+    set +o xtrace
```

Comment:
Any need to unset after we're done?

Reply:
xtrace enhances the debugging info we get from bash, it's fine to run dc stop (the last thing we do) with this still on